tests/spiffe/README.txt

```diff
@@ -0,0 +1,54 @@
+An SVID is a structured x509 certificate that follows the SPIFFE x509-SVID
+format available at:
+
+	https://github.com/spiffe/spiffe/blob/main/standards/X509-SVID.md
+
+For these tests, an SVID was created following the example from: 
+
+	https://spiffe.io/docs/latest/try/getting-started-linux-macos-x/
+
+```
+git clone https://github.com/spiffe/spire
+cd spire
+go build ./cmd/spire-server
+go build ./cmd/spire-agent
+./spire-server run -config conf/server/server.conf&
+./spire-server healthcheck
+./spire-server token generate -spiffeID spiffe://example.org/myagent
+./spire-agent run -config conf/agent/agent.conf -joinToken <token_string> &
+./spire-agent healthcheck
+./spire-server entry create -parentID spiffe://example.org/myagent \
+    -spiffeID spiffe://example.org/myservice -selector unix:uid:$(id -u)
+mkdir /tmp/spiffe-svid
+./spire-agent api fetch x509 -write /tmp/spiffe-svid
+```
+
+
+The above commands generate three files
+* `bundle.0.pem` contains the set of root CAs trusted by your current trust domain
+* `svid.0.pem` contains the SVID (x509 certificate) and its associated chain of
+  trust. In this case, the intermediate agent's own identity.
+* `svid.0.key` contains the private key of the the SVID for `myservice`
+
+You can inspect the above using
+
+```
+openssl x509 -in bundle.0.pem -text
+openssl x509 -in svid.0.pem -text # only shows the first certificate, which is the SVID
+```
```

Comment:
```suggestion
```bash
openssl x509 -in /tmp/spiffe-svid/bundle.0.pem -text
openssl x509 -in /tmp/spiffe-svid/svid.0.pem -text # only shows the first certificate, which is the SVID
```
```